source/assets/sass/config/color/_content.scss

```diff
@@ -1,10 +1,10 @@
 @use 'sass:color';
 @use 'brand';
 
-$color--highlight: color.adjust(brand.$sl-color--hopbush, $lightness: -10%);
-$color--action: color.adjust(brand.$sl-color--bouquet, $lightness: -10%);
-$color--shadow: rgba(brand.$sl-color--midnight-blue, 0.125);
-$color--active: color.adjust(brand.$sl-color--venus, $lightness: -10%);
-$color-code--bg: #f8f8f8;
-$color-code--text: color.adjust(brand.$sl-color--pale-sky, $lightness: -25%);
-$color-link--action: rgba(218, 219, 223, 25%);
+$sl-color--highlight: color.adjust(brand.$sl-color--hopbush, $lightness: -10%);
+$sl-color--action: color.adjust(brand.$sl-color--bouquet, $lightness: -10%);
+$sl-color--shadow: rgba(brand.$sl-color--midnight-blue, 0.125);
+$sl-color--active: color.adjust(brand.$sl-color--venus, $lightness: -10%);
+$sl-color--code-bg: #f8f8f8;
+$sl-color--code-text: color.adjust(brand.$sl-color--pale-sky, $lightness: -25%);
+$sl-color--link-action: rgba(218, 219, 223, 25%);
```

Comment:
I added the prefixes here so it was more consistent when these are used as Sass variables.